packages/next/lib/constants.ts

```diff
@@ -84,3 +85,8 @@ export const ESLINT_PROMPT_VALUES = [
     config: null,
   },
 ]
+
+export const PAGE_RUNTIME: Record<string, ServerRuntime> = {
```

Comment:
```suggestion
export const SERVER_RUNTIME: Record<string, ServerRuntime> = {
```
Should we rename the constant as well?